Tasks/GulpV1/typings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Ideally we are trying to move away from typings to @ types since typings is deprecated. If you want it can be a separate PR.